### PR TITLE
Clear validation errors when a resource template is loaded

### LINKED ISSUE #745: Errors not being cleared when new Resource Template is loaded
When a resource template is loaded and validation errors are in Redux following the user clicking on the **Save &amp; Publish**, if the user loads a second resource template, the errors are still present in the new form.

Screen Shot 2019-06-17 at 12 17 12 PM

Now, loading a second resource template, the errors from the first resource template are still present:

Screen Shot 2019-06-17 at 12 18 19 PM

This is likely related to https://github.com/LD4P/sinopia_editor/issues/651

### PR DESCRIPTION
**Pull request by aaron-collier**
Fixes #745 

Clears the errors and sets displayValidations to false when a resource template is loaded. Adds unit testing for the values and an integration test for displaying and clearing the validation errors on the page.

Before: After validation errors and clicking on a new template - Validation errors are still displayed. 

Screen Shot 2019-06-20 at 11 14 57 AM

After:  After validation errors and clicking on a new template - Validation errors are cleared.

Screen Shot 2019-06-20 at 11 15 28 AM

### CODE REVIEW

File: __tests__/integration/previewRDFHelper.js [hunk 1]
```diff
@@ -27,3 +27,18 @@ export async function fillInRequredFieldsForBibframeInstance() {
   await page.waitForSelector('#rbt-menu-item-0')
   await page.click('#rbt-menu-item-0')
 }
+
+export async function incompleteFieldsForBibframeInstance() {
+  // This assertion adds 1 to each it blocks assertion count
+  await pupExpect(page).toClick('a', { text: 'BIBFRAME Instance' })
+
+  // Click on one of the property type rows to expand a nested resource
```
Comment (jcoyne) [inline __tests__/integration/previewRDFHelper.js]: do we need the rest of this function?

Comment (aaron-collier) [reply __tests__/integration/previewRDFHelper.js]: Oh yeah, probably not. That's just how I was manually testing it. Will update.

Comment (aaron-collier) [reply __tests__/integration/previewRDFHelper.js]: Done. Good catch.

File: __tests__/integration/previewRDFHelper.js [hunk 2]
```diff
@@ -27,3 +27,8 @@ export async function fillInRequredFieldsForBibframeInstance() {
   await page.waitForSelector('#rbt-menu-item-0')
   await page.click('#rbt-menu-item-0')
 }
+
+export async function incompleteFieldsForBibframeInstance() {
+  // This assertion adds 1 to each it blocks assertion count
```
Comment (ndushay) [inline __tests__/integration/previewRDFHelper.js]: PR looks fine but I don't understand this comment.

Comment (aaron-collier) [reply __tests__/integration/previewRDFHelper.js]: I'm not great and clarifying this, but this is the assertion that means any block "it" block that includes this call needs to check for it's assertions+1.

Comment (ndushay) [reply __tests__/integration/previewRDFHelper.js]: ah!  how about

```
// this assertion adds 1 to each "it" block's expect.assertion(n) count
```

**Pull request merged**